src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  if (sourceString) {
+    return sourceString
+      .split(';')
+      .map((row) => row.split(':').map((param) => param.trim()))
+      .filter((row) => row.length > 1)
+      .map((e) => ({ [e[0]]: e[1] }))
+      .reduce((prew = {}, e) => ({ ...prew, ...e }));
```

Comment:
The variable naming `prew` could be improved for better readability. Consider using a name that reflects its role as an accumulator, such as `acc`, `accumulator`, or `styleObject`.